src/frontend/styles/components/page.pcss

```diff
@@ -496,3 +496,10 @@
     }
   }
 }
+
+.page_footer {
```

Comment:
Wrong BEM. 

If it's a block, name it `page-footer`.
If it's an element of the Page block, name it `page__footer`.